code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -602,21 +602,9 @@
 	var/turf/T2 = get_turf(A)
 	if(T == T2)
 		if(!silent)
-			to_chat(owner, "<span class='warning'>That's far too close!</span>")
+			to_chat(owner, "<span class='xenowarning'>That's far too close!</span>")
```

Comment:
```suggestion
			to_chat(owner, "<span class='warning'>That's far too close!</span>")
```
For failed actions `warning` is generally better.
`xenowarning` tends to be used on other contexts, such as feedback from events happening around. Would like to switch the failed actions in the long term to the generic warning red text.